plugin/core/sessions.py

```diff
@@ -1153,6 +1168,10 @@ def m_client_unregisterCapability(self, params: Any, request_id: Any) -> None:
                 if isinstance(discarded, dict):
                     for sv in self.session_views_async():
                         sv.on_capability_removed_async(registration_id, discarded)
+            if self._plugin:
+                inform = functools.partial(
+                    self._plugin.on_unregister_capability_async, registration_id, capability_path, registration_path)
+                sublime.set_timeout_async(inform)
```

Comment:
This shouldn't be done via `set_timeout_async` because we need to be informed of an unregistration before the response is sent to the language server.

I see `client/registerCapability` as a kind of constructor/`__init__`, whereas its counterpart `client/unregisterCapability` is a destructor/`__del__`, so all operations need to be done in reverse. So in `client/registerCapability` we inform the `AbstractPlugin` after informing the SessionViews. Here we should do the callback before doing the callbacks for SessionViews.